discovery-infra/test_infra/controllers/node_controllers/terraform_controller.py

```diff
@@ -64,21 +58,17 @@ def _terraform_params(self, **kwargs):
             "libvirt_dns_records": kwargs.get("dns_records", {}),
             # TODO change to namespace index
             "libvirt_network_if": self.config.net_asset.libvirt_network_if,
-            "libvirt_worker_disk": operators_utils.resource_param(
-                kwargs.get("worker_disk", 21474836480), consts.OperatorResource.WORKER_DISK_KEY, operators),
-            "libvirt_master_disk": operators_utils.resource_param(
-                kwargs.get("master_disk", 128849018880), consts.OperatorResource.MASTER_DISK_KEY, operators),
+            "libvirt_worker_disk": kwargs.get("worker_disk", 26843545601),
+            "libvirt_master_disk": kwargs.get("master_disk", 128849018880),
             "libvirt_secondary_network_name": consts.TEST_SECONDARY_NETWORK + self.config.cluster_name.suffix,
             "libvirt_storage_pool_path": kwargs.get("storage_pool_path", os.path.join(os.getcwd(), "storage_pool")),
             # TODO change to namespace index
             "libvirt_secondary_network_if": self.config.net_asset.libvirt_secondary_network_if,
             "provisioning_cidr": self.config.net_asset.provisioning_cidr,
             "running": True,
             "single_node_ip": kwargs.get("single_node_ip", ''),
-            "master_disk_count": operators_utils.resource_param(
-                kwargs.get("master_disk_count", 1), consts.OperatorResource.MASTER_DISK_COUNT_KEY, operators),
-            "worker_disk_count": operators_utils.resource_param(
-                kwargs.get("worker_disk_count", 1), consts.OperatorResource.WORKER_DISK_COUNT_KEY, operators),
+            "master_disk_count": kwargs.get("master_disk_count", 1),
```

Comment:
Here also, values are in `consts.resources`

Reply:
I restored old code, haven't checked what the default value is.